app/Main.hs

```diff
@@ -153,8 +152,62 @@ stopAndRemove (ServiceDefinition _ name buildContext buildOptions _) = do
             Left err -> liftIO $ print err
         Nothing -> return ()
 
+-- TODO: some long-running operations such as building or pulling docker image may need special care
+-- TODO: handle exceptions in main thread
```

Comment:
Prefer github issues over ToDo comments (easier to access, maintain and in general to keep track of)